src/components/Header.css

```diff
@@ -0,0 +1,43 @@
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between; 
+    background-color: white;
```

Comment:
It would be better to use a common approach to color values, e.g. only names "white", or only Hexadecimal Color Values (#ffffff).